opendevin/llm/llm.py

```diff
@@ -36,9 +54,16 @@ def my_wait(retry_state):
             opendevin_logger.info(f'Attempt #{retry_state.attempt_number} | Sleeping for {seconds}s for {retry_state.outcome.exception()}', )
             return seconds
 
+        def rate_limited_attempt(retry_state):
+            opendevin_logger.info(f'Rate limited. Waiting for {retry_state.outcome.exception()}')
+            return True
+
+        @retry(reraise=True,
+               stop=stop_after_attempt(num_retries),
+               wait=my_wait, retry=retry_if_exception_type(APIConnectionError))
```

Comment:
not related to this PR but I think it would be great if you could rename `my_wait` to a more meaningful name.